node/config/def.go

```diff
@@ -67,6 +72,10 @@ func defCommon() Common {
 				"/ip4/0.0.0.0/tcp/0",
 				"/ip6/::/tcp/0",
 			},
+
```

Comment:
Do we maybe want to protect bootstrappers by default?

Reply:
That would be good, bootstrappers can still disconnect from you, but you probably don't ever want to willingly disconnect from them?